client/core/notification.go

```diff
@@ -83,10 +83,22 @@ func (c *Core) updateCoreMatch(n *MatchNote) {
 	}
 }
 
+type NotificationFeed <-chan Notification
+
+func (notes NotificationFeed) drain() {
+	for {
+		select {
+		case <-notes:
+		default:
+			return
+		}
+	}
+}
+
 // NotificationFeed returns a new receiving channel for notifications. The
 // channel has capacity 1024, and should be monitored for the lifetime of the
 // Core. Blocking channels are silently ignored.
-func (c *Core) NotificationFeed() <-chan Notification {
+func (c *Core) NotificationFeed() NotificationFeed {
```

Comment:
I'd prefer not to make this API change since it only helps with the tests and obscures what is being returned.  Instead, how about a function in core_test.go like `drainNotes(<-chan Notification)`?

Reply:
Was thinking I'd find uses for this in the `trade_simnet_test` but that's just a test also, will refactor.

Reply:
Thanks. Part of my consideration is not just clarify, but breaking consumers for no apparent reason to them.